### DISCUSSION TITLE
Explore plugins in the form of scripts

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#147] [Ideas]
.

**Comment by U-C-S** [comment]
https://github.com/rhaiscript/rhai

Lua:
https://github.com/mlua-rs/mlua
https://github.com/NLua/NLua

**Reply by aloneguid** [reply]
I would recommend Lua to be honest, it just works. I recently had a go: https://aloneguid.github.io/bt/scripting.html
